lib/capture/response.ex

```diff
@@ -2,12 +2,12 @@ defmodule Capture.Response do
   use Ecto.Schema
 
   schema "responses" do
-    field :survey_id, :integer
-    field :question_id, :integer
-    field :strongly_disagree, :integer
-    field :disagree, :integer
-    field :neutral, :integer
-    field :agree, :integer
-    field :strongly_agree, :integer
+    field(:survey_id, :integer)
```

Comment:
I am a little confused about the brackets being added here. Was that a suggestion made by `mix format`?

Reply:
Probably yup.